src/page-objects/administration/LandingPageDetail.ts

```diff
@@ -0,0 +1,65 @@
+import type { Page, Locator } from '@playwright/test';
+import type { PageObject } from '../../types/PageObject';
+
+export class LandingPageDetail implements PageObject {
+
+    /**
+     * General
+     */
+    public readonly nameInput: Locator;
+    public readonly landingPageStatus: Locator;
+    public readonly salesChannelSelect: Locator;
+    public readonly salesChannelItem: Locator;
+    public readonly saveLandingPageButton: Locator;
+    public readonly loadingSpinner: Locator;
+    public readonly seoUrlInput: Locator;
+
+    /**
+     * Layout
+     */
+    public readonly layoutTab: Locator;
+    public readonly assignLayoutButton: Locator;
+    public readonly searchLayoutInput: Locator;
+    public readonly layoutItem: Locator;
+    public readonly layoutSaveButton: Locator;
+    public readonly layoutAssignmentCardTitle: Locator;
+    public readonly layoutAssignmentCardHeadline: Locator;
+    public readonly changeLayoutButton: Locator;
+    public readonly editInDesignerButton: Locator;
+    public readonly layoutResetButton: Locator;
+    public readonly layoutEmptyState: Locator;
+    public readonly assignCreateNewLayoutButton: Locator;
+    public readonly layoutAssignmentStatus: Locator;
+    public readonly layoutAssignmentContentSection: Locator;
+
+    constructor(public readonly page: Page) {
+        //Common
+        this.loadingSpinner = page.locator('.sw-loader');
+        this.saveLandingPageButton = page.getByRole('button', { name: 'Save' });
+        //General
+        this.nameInput = page.getByLabel('Name');
+        this.landingPageStatus = page.locator('#landingPageActive');
+        this.salesChannelSelect = page.locator('.sw-landing-page-detail-base__sales_channel .sw-select-selection-list');
+        this.salesChannelItem = page.locator('.sw-select-option--0');
```

Comment:
That is a bit hard to implement:
You have to implement it in 2 steps because in the test you also have to make 2 steps. First a click to open the selection list and the second is to choose the corresponding list item...

I recommend that way:
First locator to open the list via click() :
```
this.salesChannelSelectionList = page.locator('.sw-select')
            .filter({ hasText: 'Sales Channels' })
            .locator('.sw-select-selection-list');
```

Second you get all items of the list:
`this.filtersResultPopoverItemList = page.locator('.sw-select-result-list__content').getByRole('listitem');`

Within the test you can use then:
```
await PageObject.salesChannelSelectionList.click();
await PageObject.filtersResultPopoverItemList.getByText('0 acceptance test').click();
```